java/test/jmri/jmrit/display/layoutEditor/LayoutEditorTest.java

```diff
@@ -647,12 +649,63 @@ public void testSetDirectTurnoutControl() {
         Assert.assertTrue("getDirectTurnoutControl after set", le.getDirectTurnoutControl());
     }
 
+    @Test
+    public void testSetDirectTurnoutControlOff() {
+        Assume.assumeFalse(GraphicsEnvironment.isHeadless());
+        le.setDirectTurnoutControl(false);
+        Assert.assertFalse("getDirectTurnoutControl after set", le.getDirectTurnoutControl());
+    }
+
+    @Test
+    public void testIsEditableDefault() {
+        Assume.assumeFalse(GraphicsEnvironment.isHeadless());
+        // default to true
+        Assert.assertTrue("isEditable default true", le.isEditable());
+    }
+
+    @Test
+    public void testSetAllEditableFalse() {
+        Assume.assumeFalse(GraphicsEnvironment.isHeadless());
+        le.setAllEditable(false);
+        Assert.assertFalse("isEditable after setAllEditable(false)", le.isEditable());
+    }
+
+    @Test
+    public void testSetAllEditableTrue() {
+        Assume.assumeFalse(GraphicsEnvironment.isHeadless());
+        le.setAllEditable(true);
+        Assert.assertTrue("isEditable after setAllEditable(true)", le.isEditable());
+    }
+
+    @Test
+    public void testGetHighlightSelectedBlockDefault() {
+        Assume.assumeFalse(GraphicsEnvironment.isHeadless());
+        // default to false
+        Assert.assertFalse("le.getHighlightSelectedBlock default false", le.getHighlightSelectedBlock());
+    }
+
+    @Test
+    public void testSetHighlightSelectedBlockTrue() {
+        Assume.assumeFalse(GraphicsEnvironment.isHeadless());
+        le.setHighlightSelectedBlock(true);
+        Assert.assertTrue("le.getHighlightSelectedBlock after setHighlightSelectedBlock(true)", le.getHighlightSelectedBlock());
+    }
+
+    @Test
+    public void testSetHighlightSelectedBlockFalse() {
+        Assume.assumeFalse(GraphicsEnvironment.isHeadless());
+        le.setHighlightSelectedBlock(false);
+        Assert.assertFalse("le.getHighlightSelectedBlock after setHighlightSelectedBlock(false)", le.getHighlightSelectedBlock());
+    }
+
     // from here down is testing infrastructure
     @Before
     public void setUp() throws Exception {
-        JUnitUtil.setUp();
-        if(!GraphicsEnvironment.isHeadless()){
-           le = new LayoutEditor("Test Layout");
+        apps.tests.Log4JFixture.setUp();
+        // reset the instance manager.
+        JUnitUtil.resetInstanceManager();
```

Comment:
Lines 704-706 above reverse @rhwood's recent change here.  Those need to be reverted to The call to JUnitUtil.setUp()

Reply:
Done!

Reply:
That change is still showing as it did before.  Did you remember to commit the file?

Reply:
Weird... don't remember de-selecting that one... don't know how it got missed. Probably forgot to save in NetBeans after I made the change.